HLTrigger/Configuration/python/HLT_75e33/modules/hltPhase2SiPixelClustersSoA_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+hltPhase2SiPixelClustersSoA = cms.EDProducer(
+    "SiPixelPhase2DigiToCluster@alpaka",
+    alpaka = cms.untracked.PSet(
+        backend = cms.untracked.string('')
+    )
```

Comment:
Pedantic suggestion, for consistency with the other modules:
```suggestion
hltPhase2SiPixelClustersSoA = cms.EDProducer("SiPixelPhase2DigiToCluster@alpaka",
    # autoselect the alpaka backend
    alpaka = cms.untracked.PSet(backend = cms.untracked.string(''))
```